rotary_jet_spinning/models/modelling.py

```diff
@@ -34,7 +34,8 @@ def critical_rotational_velocity_threshold(self, surface_tension, orifice_radius
         
     def Initial_velocity(self, omega_th, s0):
```

Comment:
small error in the name. It should not be a capital first later.

Reply:
I put a capital to the name of the function because I already used initial_velocity as a variable name. Should I change the name of the function by initial_velocity_fct for example or can I keep the capital letter Initial_velocity ?

Reply:
Ok that is a conflict we shoud resolve. Are both of them referring to the same thing? If not, one of them may be not well named.
Is the variable initialized using the function? If so, what about changing the variable name to v_ini

Reply:
yes there are referring to the same thing, the variable is initialized with the function yes. ok I changed the variable name to init_velocity